src/test/javascript/portal/filter/DateFilterSpec.js

```diff
@@ -55,11 +55,6 @@ describe("Portal.filter.DateFilter", function() {
             expect(filter.getCql()).toBe("column_name >= '1999-01-01T00:00:00Z'");
         });
 
-        it('gives data layer CQL', function() {
```

Comment:
These are fairly important unit tests which no longer appeared to be covered anywhere (as far as I can see)?